feat: implmenet new Banner design, follow the new block production animation effect

WhiteMinds opened the pull request:
feat: implmenet new Banner design, follow the new block production animation effect

Ref: https://github.com/Magickbase/ckb-explorer-public-issues/issues/175

Keith-CY commented:
The count of random protruding pillars is expected to equal to transaction count in the latest block, as mentioned in point 4 in https://github.com/Magickbase/visual-style-guide/issues/22#issuecomment-1407319332

The resolution is a bit low, and `CKB` is not centered in mobile layout
![image](https://user-images.githubusercontent.com/7271329/218523019-f2e65241-765e-4d6f-9fd4-db365080718f.png)

WhiteMinds commented:
> The count of random protruding pillars is expected to equal to transaction count in the latest block

Changed.

> The resolution is a bit low, and CKB is not centered in mobile layout

Text centered, about the resolution, I now changed to the layout of the mobile device to 600 * 150, looks the effect will be better.

> The protruding pillars are a bit higher than the static CKB
They are expected to be high as CKB

@Keith-CY I don't understand these two sentences clearly. It seems that the maximum height of pillars needs to be adjusted to the height when CKB is static?
It was 120 before, and I now set it to 40, maybe 60 is better?

I added some redundant commits for review.
Before merging, please remind me to do a rebase to merge these commits.

Keith-CY commented:
This PR is going to be merged, the further optimization could be added based on the develop branch
